README.md

```diff
@@ -55,7 +68,7 @@ Describe the importance of using object-oriented programming.
 
 Your Answer:
 ```js
-// your answer here
+// Using OOP has many benefits and implications for pragmatic programming best practices. With OOP, an object is created and the information used to create it no longer has to be reused again, leading to increased productivity, less errors, and more flexibility in creating new functionality. Also, OOP programs are much easier to modify and maintain.
```

Comment:
:+1:
In the future, please discuss encapsulation and representing real world objects.